man/systemd.mount.xml

```diff
@@ -497,6 +497,17 @@
         off.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>ReadWriteOnly=</varname></term>
+
+        <listitem><para>Takes a boolean argument. If true, a failed
+        read-write mount is not retried with a read-only mount.
```

Comment:
not liking the inverted sentence here. Maybe say:

```
If false, a mount point that shall be mounted read-write but cannot be mounted so is retried to be mounted read-only. If true the operation will fail immediately after the read-write mount attempt did not succeed.
```

Or maybe @keszybz has some more ideas?